2021-02-25 15:03:38,085 INFO    MainThread:13670 [wandb_setup.py:_flush():69] setting env: {}
2021-02-25 15:03:38,085 INFO    MainThread:13670 [wandb_setup.py:_flush():69] setting login settings: {}
2021-02-25 15:03:38,085 INFO    MainThread:13670 [wandb_setup.py:_flush():69] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f07f7ab2090>
2021-02-25 15:03:38,085 INFO    MainThread:13670 [wandb_init.py:_log_setup():319] Logging user logs to /home/dacon/Dacon/bdg/ct/segment_abdomen/src/wandb/run-20210225_150338-17dux9gg/logs/debug.log
2021-02-25 15:03:38,085 INFO    MainThread:13670 [wandb_init.py:_log_setup():320] Logging internal logs to /home/dacon/Dacon/bdg/ct/segment_abdomen/src/wandb/run-20210225_150338-17dux9gg/logs/debug-internal.log
2021-02-25 15:03:38,085 INFO    MainThread:13670 [wandb_init.py:init():352] calling init triggers
2021-02-25 15:03:38,085 INFO    MainThread:13670 [wandb_init.py:init():359] wandb.init called with sweep_config: {}
config: {'path': '../data/normal2', 'name': 'timm-efficientnet-b7', 'model': 'timm-efficientnet-b7', 'pretrain': 'noisy-student', 'model_path': '../models/timm-efficientnet-b7', 'augmentation': [], 'num_epochs': 20, 'batch_size': 8, 'optimizer': <class 'torch.optim.adam.Adam'>, 'scheduler': <class 'torch.optim.lr_scheduler.CosineAnnealingWarmRestarts'>, 'loss': <class 'segmentation_models_pytorch.utils.losses.DiceLoss'>, 'seed': 1015, 'device': 'cuda:0'}
2021-02-25 15:03:38,085 INFO    MainThread:13670 [wandb_init.py:init():374] re-initializing run, found existing run on stack: 2nqutbko
2021-02-25 15:03:38,087 INFO    MainThread:13670 [wandb_run.py:finish():1078] finishing run deagwon-bu/ct_segment/2nqutbko
2021-02-25 15:03:38,087 INFO    MainThread:13670 [wandb_init.py:_jupyter_teardown():250] cleaning up jupyter logic
2021-02-25 15:03:38,087 INFO    MainThread:13670 [jupyter.py:save_history():208] not saving jupyter history
2021-02-25 15:03:38,087 INFO    MainThread:13670 [wandb_run.py:_atexit_cleanup():1392] got exitcode: 0
2021-02-25 15:03:38,087 INFO    MainThread:13670 [wandb_run.py:_restore():1364] restore
2021-02-25 15:03:38,376 INFO    MainThread:13670 [wandb_run.py:_wait_for_finish():1515] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 854
  total_bytes: 854
}

2021-02-25 15:03:40,390 INFO    MainThread:13670 [wandb_run.py:_wait_for_finish():1515] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5037
  total_bytes: 5037
}

2021-02-25 15:03:41,715 INFO    MainThread:13670 [wandb_run.py:_show_files():1737] logging synced files
2021-02-25 15:03:41,722 INFO    MainThread:13670 [wandb_init.py:init():401] starting backend
2021-02-25 15:03:41,723 INFO    MainThread:13670 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-02-25 15:03:41,728 INFO    MainThread:13670 [backend.py:ensure_launched():123] starting backend process...
2021-02-25 15:03:41,778 INFO    MainThread:13670 [backend.py:ensure_launched():128] started backend process with pid: 13823
2021-02-25 15:03:41,779 INFO    MainThread:13670 [wandb_init.py:init():406] backend started and connected
2021-02-25 15:03:41,790 INFO    MainThread:13670 [wandb_init.py:init():446] updated telemetry
2021-02-25 15:03:41,791 INFO    MainThread:13670 [wandb_init.py:init():465] communicating current version
2021-02-25 15:03:42,403 INFO    MainThread:13670 [wandb_init.py:init():470] got version response 
2021-02-25 15:03:42,403 INFO    MainThread:13670 [wandb_init.py:init():478] communicating run to backend with 30 second timeout
2021-02-25 15:03:42,738 INFO    MainThread:13670 [wandb_init.py:init():503] starting run threads in backend
2021-02-25 15:03:43,728 INFO    MainThread:13670 [wandb_run.py:_console_start():1422] atexit reg
2021-02-25 15:03:43,728 INFO    MainThread:13670 [wandb_run.py:_redirect():1285] redirect: SettingsConsole.WRAP
2021-02-25 15:03:43,729 INFO    MainThread:13670 [wandb_run.py:_redirect():1320] Wrapping output streams.
2021-02-25 15:03:43,729 INFO    MainThread:13670 [wandb_run.py:_redirect():1336] Redirects installed.
2021-02-25 15:03:43,729 INFO    MainThread:13670 [wandb_init.py:init():527] run started, returning control to user process
2021-02-25 15:04:15,204 INFO    MainThread:13670 [wandb_run.py:_atexit_cleanup():1392] got exitcode: 0
2021-02-25 15:04:15,204 INFO    MainThread:13670 [wandb_run.py:_restore():1364] restore
2021-02-25 15:04:15,513 INFO    MainThread:13670 [wandb_run.py:_wait_for_finish():1515] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 854
  total_bytes: 854
}

2021-02-25 15:04:17,526 INFO    MainThread:13670 [wandb_run.py:_wait_for_finish():1515] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5234
  total_bytes: 5234
}

2021-02-25 15:04:18,846 INFO    MainThread:13670 [wandb_run.py:_show_files():1737] logging synced files
